docs/howto/operate/move-hub.md

```diff
@@ -23,29 +23,67 @@ home directory storage!
 
 Primarily used with GKE right now.
 
-1. In the target NFS server, create a new ssh key-pair, with
+1. SSH into **both** the target server and source server and then enable the `ubuntu` user on **both** servers with `sudo su ubuntu`.
+   This will ensure that the keys we are about to create will be assigned to the correct user.
+2. In the target NFS server, create a new ssh key-pair, with
```

Comment:
Don't you need to set up some specific permissions after creating the key?

Reply:
No, by making sure you activate the `ubuntu` user first and using the `ssh-keygen` command, all the permissions are taken care of.